ui/pages/institutional/custody/custody.js

```diff
@@ -208,60 +227,56 @@ const CustodyPage = () => {
 
   useEffect(() => {
     const fetchConnectRequest = async () => {
-      const connectRequestValue = await dispatch(
```

Comment:
Improvements made:
1. Removed `connectRequestValue` in favor of the one comment above
2. Handled errors in the fetchConnectRequest function itself instead of only in the handleFetchConnectRequest function. This way, if an error occurs while fetching the connect request, it can be handled immediately and appropriately.
3. USe of the finally method of the Promise returned by the fetchConnectRequest function to set the loading state to false after the fetchConnectRequest function has been completed.
4.  Instead of jwt, changed it to custodianToken to clarify what the variable represents.